packages/base-metaocaml-ocamlfind/base-metaocaml-ocamlfind.base/opam

```diff
@@ -1,14 +1,15 @@
 opam-version: "2.0"
 maintainer: "yallop@gmail.com"
 authors: ["Jeremy Yallop"]
+license: "MIT"
 depopts: ["ocamlfind"]
 homepage: "https://github.com/ocaml/opam-repository/tree/master/packages/base-metaocaml-ocamlfind/base-metaocaml-ocamlfind.base"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 install: [
   ["mkdir" "-p" "%{lib}%/findlib.conf.d/"] {ocamlfind:installed}
   ["ocamlfind" "install" "metaocaml" "META"] {ocamlfind:installed}
   ["cp" "metaocaml.native.conf" "%{lib}%/findlib.conf.d/metaocaml.conf"]
-    {ocamlfind:installed & (ocaml:compiler = "4.04.0+BER" | ocaml:compiler = "4.07.1+BER" | ocaml:compiler = "4.11.1+BER" | ocaml:compiler = "4.11.1+BER+flambda")}
+    {ocamlfind:installed & (ocaml:compiler = "4.04.0+BER" | ocaml:compiler = "4.07.1+BER" | ocaml:compiler = "4.11.1+BER" | ocaml:compiler = "4.11.1+BER+flambda" | ocaml:compiler = "4.14.1+BER+flambda")}
```

Comment:
this should include `| ocaml-compiler = "4.14.1+BER"` too ?
otherwise `ocamlfind -toolchain metocaml ocamlopt` is regular ocamlopt

Reply:
Good catch: this should read `"4.14.1+BER"`, not `"4.14.1+BER+flambda"`

Reply:
out of curiousity - why metocaml is packaged as separate ocamlfind toolchain as opposed to being the default toolchain in the switch? is it because of the potential syntax incompatilbity with `>.` operator?

Reply:
makes sense, but also a little sad (harder to use)